src/app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const { join } = require('path');
+const cookiePerser = require('cookie-parser')
+const rauter = require('./router');
+const { e404, serverErr } = require('./controllers/err');
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use(express.static(join(__dirname, '..', 'public')));
+app.use(cookiePerser);
+// app.use(rauter);
```

Comment:
remove comment line

Reply:
actually you should set up the router file

Reply:
ill do it but i need to work with Muhammed with the database queries to make it right

Reply:
I want to work on the router when Muhammed is done with the database queries